drivers/ethernet/eth_adin2111.c

```diff
@@ -979,4 +982,33 @@ static const struct ethernet_api adin2111_port_api = {
 	ADIN2111_PORT_DEVICE_INIT_INSTANCE(inst, 0, 1)						\
 	ADIN2111_PORT_DEVICE_INIT_INSTANCE(inst, 1, 2)
 
+#define ADIN1110_MAC_INITIALIZE(inst)								\
```

Comment:
The definition is pretty identical to the ADIN2111 with the exception of a port. Would it make sense to a more direct definition assignment to reduce code populated here. Also, how would this code operate if the user were to deploy two different PHY devices. It looks like the definitions are sharing the same resources, etc